helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -153,6 +153,8 @@
 
   public final static String TASK_QUOTA_RATIO_NOT_SET = "-1";
 
+  public static final long OFFLINE_TIMEOUT_FOR_PURGE_NOT_SET = -1L;
```

Comment:
nit: offline "duration"

Reply:
Actually I changed all other places to offline timeout. I feel `timeout` could better show the meaning for this duration.

Reply:
i add some more explanation here: https://github.com/apache/helix/pull/1494/files#r515667749

ultimately it's up to you to make the call since you're the owner, but i wanted to make it clear why i think using 'timeout' to describe this could be confusing.